ehr/test/src/org/labkey/test/tests/ehr/AbstractEHRTest.java

```diff
@@ -401,8 +401,9 @@ protected void initCreatedProject() throws Exception
 
         doExtraPreStudyImportSetup();
 
-        populateInitialData();
         defineQCStates();
+        populateInitialData();
```

Comment:
I take it that the ordering is important here. If so, adding a comment to capture that would be nice.

Reply:
yeah need QC states before populating data